Spotishell/Public/Restore-Library.ps1

```diff
@@ -0,0 +1,91 @@
+<#
+    .SYNOPSIS
+        Restore Library items from json file (Playlists, FollowedArtists, SavedAlbums, SavedShows, SavedTracks)
+    .EXAMPLE
+        PS C:\> Restore-Library -Path $FilePath
+        Restore all Library items from '.\mySpotifyBackup.json'
+    .EXAMPLE
+        PS C:\> Restore-Library -Type FollowedArtists,SavedAlbums -Path $FilePath
+        Restore Followed Artists and Saved Albums items from '.\mySpotifyBackup.json'
+    .PARAMETER Path
+        Path of the backup file you want to restore
+    .PARAMETER Type
+        One or more items type to restore (All, FollowedArtists, SavedAlbums, SavedShows, SavedTracks)
+    .PARAMETER ApplicationName
+        Specifies the Spotify Application Name (otherwise default is used)
+#>
+function Restore-Library {
+    param (
+        [Parameter(Mandatory)]
+        [string]
```

Comment:
```suggestion
        [ValidateScript({
            if( -Not ($_ | Test-Path) ){
                throw "File or folder does not exist"
            }
            return $true
        })]
```

Reply:
The first line of this function is Get-Content which will throw the right Exception if there is a problem with $Path (like missing file)